### DISCUSSION TITLE
Money (INR,EUR,USD)

### DISCUSSION THREAD

**Discussion by punyapalshah** [filamentphp/filament#3271] [Help] [answered]
See I think there is a bug in showing the money format of the TextColumn. if this is not and bug please guide why this is like this?

below is in EUR. //this looks correct.
![image](https://user-images.githubusercontent.com/99394531/181437895-3c8a9f43-81ec-48d2-ba65-15af8bba212d.png)

below is USD. //in this and INR they divide by 100? why?
![image](https://user-images.githubusercontent.com/99394531/181438482-eec6a898-5e9f-40f2-a8bc-99e081d1595e.png)

below is INR.
![image](https://user-images.githubusercontent.com/99394531/181438589-ee4e10ea-d765-4b3b-9d08-d5f6d93cc85c.png)

**Comment by danharrin** [comment, marked as answer]
Each of those currencies have different denominations, so divide by different values. We use [this package](https://github.com/akaunting/laravel-money) for formatting. If you don't want that conversion, pass `true` as the second parameter of the column.
